libraries/provider_hab_package.rb

```diff
@@ -155,7 +148,9 @@ def current_versions
         end
 
         def installed_version(ident)
-          hab('pkg', 'path', ident).stdout.chomp.split('/')[-2..-1].join('/')
+          # Handling windows vs linux pathing
+          seperator = platform_family?('windows') ? '\\' : '/'
```

Comment:
should be spelled separator.  also this can be replaced with `File.ALT_SEPARATOR`.

you might consider this, although I'm not certain its any clearer:

```
[7] pry(main)> file = "/foo/bar/baz/fizz/buzz"
=> "/foo/bar/baz/fizz/buzz"
[8] pry(main)> File.join(File.basename(File.dirname(file)), File.basename(file))
=> "fizz/buzz"
```

File.basename + File.dirname should DTRT when it comes to the ALT_SEPARATOR on windows.

Reply:
During my rebase this bit of code got replaced by changes from #148